Purchasing.Mvc/AutomapperInstaller.cs

```diff
@@ -1,22 +1,42 @@
-﻿using AutoMapper;
+﻿using System.Linq;
+using AutoMapper;
 using Purchasing.Core.Domain;
 using Purchasing.Core.Queries;
-using Purchasing.Mvc.Controllers;
+using Castle.MicroKernel.Registration;
+using Castle.MicroKernel.SubSystems.Configuration;
+using Castle.Windsor;
 
 namespace Purchasing.Mvc
 {
-    public class AutomapperConfig
+    public class AutoMapperInstaller : IWindsorInstaller
     {
-        public static void Configure()
+        public void Install(IWindsorContainer container, IConfigurationStore store)
         {
-            Mapper.Initialize(cfg => cfg.AddProfile<ViewModelProfile>());
+            // Register all mapper profiles
+            container.Register(
+                Classes.FromAssemblyInThisApplication(GetType().Assembly)
+                .BasedOn<Profile>().WithServiceBase());
```

Comment:
Profile? Should this be ViewModelProfile ?

Reply:
Yes. Good catch.

Reply:
NVM, this is how the Automapper docs say to do it.